resolver/init.sls

```diff
@@ -2,9 +2,16 @@
 ##### Salt Formula For Resolver #####
 #####################################
 
+{% set resolvconf = salt['pkg.version']('resolvconf') %}
+
 # Resolver Configuration
-/etc/resolv.conf:
+resolv-file:
   file.managed:
+    {% if resolvconf %}
+    - name: /etc/resolvconf/resolv.conf.d/base
```

Comment:
I couldn't find anything about resolvconf on other non Debian/Ubuntu distros. What's the best approach to check for resolvconf support?

On Ubuntu, package provides a few network up/down hooks to run "resolvconf -u", we can check for package or the hook script path, or I can add a "grains.os == Ubuntu" check to line 5.